dali/base/dasds.cpp

```diff
@@ -5323,19 +5351,15 @@ class CStoreHelper : implements IStoreHelper, public CInterface
 
         OwnedIFile iFile = createIFile(path.str());
         OwnedIFileIO iFileIO = iFile->open(IFOcreate);
-        if (crcXml)
-        {
-            iFileIO->write(0, sizeof(unsigned), crcXml);
-            if (crcBinary)
-                iFileIO->write(sizeof(unsigned), sizeof(unsigned), crcBinary);
-        }
-        if (storeInfo)
-            storeInfo->cache.set(filename.str());
+        storeInfo->serialize(iFileIO, crcXml, crcBinary);
```

Comment:
this is/was not doing a "serialize"..
It was writing the newly calculated crc to a new store file (and it was not updating storeInfo).
So name misleading, and should not change semantics - i.e. it is now altering the members of CSstoreInfo here, where it did not do before. 

It *might* be okay to change the semantics, alter storeInfo, then serialize, but in that case it should be written differently anyway, and looks like an accidental change in semantics.

To preserve the previous behaviour, I suggest calling it 'save' and make it a static, since it's should not be using the members.